### PR TITLE
Updating Colorways Part I

### PR DESCRIPTION
**Pull request by xplusplus**
## Description
I decided to add some recent colorways to the configurator. I'm pulling from the [MechMadness 2020 Designs](https://mechmadness2020.com/designs) list, with the hopes of adding the entire list before the Keyset Final in early April. 

## Approved Sets in this PR:
- GMK 8008
    - Permission from Dixie granted
- GMK 9009 (R3)
    - Permission* from u/janglad granted
- GMK Analog Dreams
    - Permission from pwade3 granted
- GMK Ascii
- GMK Calm Depths
    - Permission from Oblotzky granted
- GMK Bingsu
    - Permission from ProjectKeyboard granted
- GMK Cafe
    - Permission from Langelandia granted

*9009 uses vintage stock colors

### Other work
- Removed some redundant SCSS from GMK Bento

## Testing
- Visually confirmed 40% ortholinear, 60%, 65%, TKL, and 1800.

### CODE REVIEW

**Comment by yanfali** [comment]
This is pretty neat. Did you get permission from the designers? We ask the designer for their approval before merging colorways. For example Mito has refused us in the past so we respect his choice and do not add any of his designs.

**Comment by xplusplus** [comment]
Ah I see. In that case, I will hold off on this PR and gather written permission for proceeding any further. Thanks for the heads up @yanfali .

**Comment by oblotzky** [comment]
Permission granted for any of the sets I do.

**Comment by yanfali** [comment]
@xplusplus cut and paste notes you get from designers into bug if it's not too much trouble. that way we have a audit trail.

File: src/scss/style.scss [hunk 1]
```diff
@@ -482,6 +482,9 @@ button {
 .right-side {
   float: left;
   pointer-events: all;
+  p {
+    height: 22px;
+  }
```
Comment (noroadsleft) [inline src/scss/style.scss]: > Edit: also added fix for height issue I ran into on Chromium Edge (80.0.361.62 64-bit)

Looking at this CSS, it looks like you ran into a bug that we patched a couple days ago.

https://github.com/qmk/qmk_configurator/pull/666

If the [currently-deployed QMK Configurator](https://config.qmk.fm/) renders okay for you, the changes to this file should be reverted.

**Comment by xplusplus** [comment]
@yanfali, @skullydazed preferred I do not track consent via a bug and closed it, just FYI.

**Comment by yanfali** [comment]
@xplusplus I actually expected you to re-use this bug so I'm not all that shocked that @skullydazed was not kosher with that.

**Comment by skullydazed** [comment]
Yes, we want permission tracked here with the PR so that if there's any question later we don't have to go searching through multiple issues.

**Comment by projectkeyboard** [comment]
Permission granted. Thanks for doing this!

File: src/components/colorways/gmk/index.js [hunk 2]
```diff
@@ -1,4 +1,13 @@
+import gmk_8008 from './8008';
+import gmk_9009 from './9009';
+import gmk_analog_dreams from './analog_dreams';
+import gmk_ascii from './ascii';
 import gmk_bento from './bento';
+import gmk_bingsu from './bingsu';
+import gmk_cafe from './cafe';
+import gmk_calm_depths from './calm_depths';
+// import gmk_camping from './camping';
+// import gmk_coral from './coral';
```
Comment (noroadsleft) [inline src/components/colorways/gmk/index.js:10]:
```suggestion
```

File: src/components/colorways/gmk/index.js [hunk 3]
```diff
@@ -17,7 +26,16 @@ import gmk_wob from './wob';
...
+  gmk_8008,
+  gmk_9009,
+  gmk_analog_dreams,
+  gmk_ascii,
   gmk_bento,
+  gmk_bingsu,
+  gmk_cafe,
+  gmk_calm_depths,
+  // gmk_camping,
+  // gmk_coral,
```
Comment (noroadsleft) [inline src/components/colorways/gmk/index.js:38]:
```suggestion
```

File: src/scss/colorways.scss [hunk 4]
```diff
@@ -293,19 +355,71 @@
...
+// .gmk-coral-mod {
+//   background: #fa7268;
+//   color: $color-gmk-abs-CP;
+// }
+
+// .gmk-coral-accent {
+//   background: #98dddf;
+//   color: #fa7268;
+// }
+
```
Comment (noroadsleft) [inline src/scss/colorways.scss:422]:
```suggestion
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**